wordpress-mobile/WordPress-iOS
# Edit Comment: make inline editable cells generic

Ref: #17000

Since the inline editable cells that were added for the new Edit Comment view are not specific to Comments, this makes them generic.
- `EditCommentMultiLineCell` is now `InlineEditableMultiLineCell`.
- `EditCommentSingleLineCell` is now `InlineEditableSingleLineCell`.
- The files now live in the ViewRelated > Cells directory.

To test:
- Go to My Site > Comments > Comment details > Edit.
- Edit a single line value and the comment content.
- Select `Done`.
- Verify the comment is updated accordingly.

## Regression Notes
1. Potential unintended areas of impact
The comment isn't updated correctly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Updated comment to ensure it works.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
